system/Autoloader/FileLocator.php

```diff
@@ -119,6 +119,10 @@ public function locateFile(string $file, ?string $folder = null, string $ext = '
      */
     public function getClassname(string $file): string
     {
+        if(is_dir($file)) {
+            return '';
+        }
```

Comment:
I think this code is okay (if the coding style is fixed), because a directory is never a class.